services/bugmon/launch.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+function retry () {
+  for _ in {1..9}; do
+    "$@" && return
+    sleep 30
+  done
+  "$@"
+}
+
+function tc-get-secret () {
+  TASKCLUSTER_ROOT_URL="${TASKCLUSTER_PROXY_URL-$TASKCLUSTER_ROOT_URL}" retry taskcluster api secrets get "project/fuzzing/$1"
+}
+
+git init bugmon-tc
+cd bugmon-tc
+git remote add origin "${GIT_REPO-git@bugmon-tc:MozillaSecurity/bugmon-tc}"
+retry git fetch -q --depth=10 origin "${GIT_BRANCH-master}"
+git -c advice.detachedHead=false checkout "${GIT_REVISION-origin/master}"
```

Comment:
Do you need to install bugmon-tc for the commands to be available below? It seems like all of the deps will be missing until runtime too, which is going to make the container startup much slower.